### DISCUSSION TITLE
[General] What repo is the npm package maintained from?

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [zkSync-Community-Hub/zksync-developers#490] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

This one:
https://www.npmjs.com/package/@matterlabs/zksync-contracts

It looks like it's not this, because the folder contents are different. 
https://github.com/matter-labs/era-contracts

### What is the specific issue or error you're encountering?

see above

### Can you share the error messages or logs you're receiving, if any?

see above

### Have you made any recent changes to the contract before encountering this issue?

see above

### Are there any external libraries or contracts that your contract interacts with?

see above

### Can you provide the relevant portions of your contract code where the issue is occurring?

see above

### Have you tried to isolate the problem, and if so, what were the results?

see above

### What steps have you already taken to try to resolve the issue?

see above

**Comment by bxpana** [comment]
Double checking on this

**Comment by dutterbutter** [comment, marked as answer]
The repo is https://github.com/matter-labs/era-contracts but there is a need to re-publish the npm package to reflect changes.
